### DISCUSSION TITLE
Update tutorial docs screenshot to better reflect what is in the code

### DISCUSSION THREAD

**Discussion by Ed1123** [fastapi/fastapi#8496] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI

app = FastAPI()


@app.get('/items/{item_id}')
async def read_item(item_id: int):
    return {'item': item_id}
```

### Description

In the [Tutorial](https://fastapi.tiangolo.com/tutorial/path-params/#documentation) it is shown that there's no Example Value:
<img width="855" alt="image" src="https://user-images.githubusercontent.com/44955050/169926900-66fe09a0-abf2-409b-8294-e640b0b96a4e.png">
while, when running the API locally there is that field:
<img width="711" alt="image" src="https://user-images.githubusercontent.com/44955050/169927153-008b0113-3dc5-4ede-8c51-1f046b4d6f94.png">

1. Updating the screenshot will help better reflect what exactly is displayed (I know I'm being kind of perfectionistic).
2. This is a related issue. One may expect to get the schema of the dict that is being returned in the function instead of just "string".  Leaving it blank would be more indicative, since "string" suggest the API returns only a string.

### Operating System

macOS

### Operating System Details

MacOS Monterey

### FastAPI Version

0.78.0

### Python Version

3.9.12

**Comment by ruiqurm** [comment]
For the second problem, you can add a `response_model`。
```python
class Item(pydantic.BaseModel):
    item_id : int

@app.get('/items/{item_id}',response_model=Item)
async def read_item(item_id: int):
    return {'item': item_id}
```
![image](https://user-images.githubusercontent.com/24454448/169946219-fcfe196d-7cc0-4299-ab44-e3ea237e73a3.png)

**Comment by Ed1123** [comment]
> For the second problem, you can add a `response_model`。
> ```python
> class Item(pydantic.BaseModel):
>     item_id : int
> 
> @app.get('/items/{item_id}',response_model=Item)
> async def read_item(item_id: int):
>     return {'item': item_id}
> ```
> ![image](https://user-images.githubusercontent.com/24454448/169946219-fcfe196d-7cc0-4299-ab44-e3ea237e73a3.png)
> 

Technically the result is incorrect since the JSON will have an "item_id" key instead of an "item", but it is a good point.
